doc/mgr/dashboard.rst

```diff
@@ -319,6 +319,54 @@ You need to tell the dashboard on which url Grafana instance is running/deployed
 The format of url is : `<protocol>:<IP-address>:<port>`
 You can directly access Grafana Instance as well to monitor your cluster.
 
+Enabling Single Sign-On (SSO)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+  Ceph Dashboard SSO support relies on **onelogin python-saml** library, so you have to guarantee that this library is installed on your system beforehand.
+
+  **Python 2:**
+
+  ``$ pip2 install python-saml``
+
+  **Python 3:**
+
+  ``$ pip3 install python3-saml``
+
+Ceph Dashboard supports **SAML 2.0** protocol to integrate with an existing Identity Provider (IdP).
+
+To configure SSO on Ceph Dashboard, you should use the following command::
+
+  $ ceph dashboard sso-saml2-setup <ceph_dashboard_base_url> <idp_metadata> {<idp_username_attribute>} {<idp_entity_id>} {<sp_x_509_cert>} {<sp_private_key>}
+
+Parameters:
+
+- **<ceph_dashboard_base_url>**: Base URL where ceph dashboard is accessible (e.g., `https://cephdasboard.local`)
```

Comment:
Can't the dashboard module obtain the `base_url` from the Mgr module directly? Isn't this what the output of `ceph mgr services` already returns?

Reply:
Small typo: "https://cephdasboard.local"

Reply:
Mgr module is not aware of the DNS / apache configuration that may be used to access ceph dashboard.